data-model/annotations/schema/annotation.json

```diff
@@ -3,45 +3,77 @@
   "type": "object",
```

Comment:
Suggestion to add a title to every JSON schema on the top level.
Gave the other models in my local copy also titles so the FE can use that to automatically generate type files (for the naming).